### ISSUE TITLE
[4] Media Manager Accessibility Issues

### ISSUE THREAD

**Issue by drmenzelit** [joomla/joomla-cms#37117]
### Steps to reproduce the issue

Go to Media Manager and try to use it with **keyboard only**

**Problems detected** 

- [ ] It is impossible to navigate inside the folders tree (tabindex=-1)
- [ ] Poor / incomplete focus visibility on elements

**Firefox**
![2022-02-22 14_15_36-Greenshot](https://user-images.githubusercontent.com/9153168/155167565-a420e46c-4179-4b86-a329-657dcb5676ee.jpg)
![2022-02-22 14_14_30-Greenshot](https://user-images.githubusercontent.com/9153168/155167705-dcd4da49-0873-486e-8dae-efdc15745510.jpg)

**Chrome**
![2022-02-22 14_17_07-Greenshot](https://user-images.githubusercontent.com/9153168/155167597-7a6dd453-5942-4b2f-936a-adddd87537ec.jpg)
![2022-02-22 14_16_22-Greenshot](https://user-images.githubusercontent.com/9153168/155167724-d059bace-d7bd-4ec2-a0bb-24aaa3b5e9a3.jpg)

- [ ] No focus visible at all in the grid
- [ ] Elements in the grid are not interactive, it is not possible to open a folder

![2022-02-22 15_11_38-Greenshot](https://user-images.githubusercontent.com/9153168/155167883-3a38685d-7f7d-413e-9038-b0fd6204f507.jpg)

- [ ] When / if you reached a folder or image on the grid, you can click with the arrow keys and get the functions visible. They don't dissapear if you tab further

![2022-02-22 14_19_17-Greenshot](https://user-images.githubusercontent.com/9153168/155168883-a9386ee2-213c-4fca-ac25-5a4fdce1af53.jpg)

- [ ] No focus on modals -> PR https://github.com/joomla/joomla-cms/pull/36892

### Expected result
The Media Manager should be accessible for all kind of users.

### Actual result
It is not possible to use the Media Manager with keyboard only.

### System information (as much as possible)

Joomla 4.1.1-dev
Windows PC
Firefox and Chrome (also screen reader extension tested)

### Additional comments
https://levelaccess.com/creating-an-accessible-aria-tree-control
https://dequeuniversity.com/library/aria/tree-view
https://vuescript.com/best-tree-view/

**Comment by akshitrattan** [comment]
@drmenzelit could you please test the PR #37152

**Comment by Quy** [comment]
Please test PR #38126.

**Comment by chmst** [comment]
Closing as we have a PR
